index.js

```diff
@@ -403,8 +403,12 @@ class Hyperbee extends ReadyResource {
     }
   }
 
-  _open () {
-    return this.core.ready()
+  async _open () {
+    await this.core.ready()
+
+    const baseCache = Rache.from(this.core.globalCache)
+    this._keyCache = new Cache(baseCache)
+    this._nodeCache = new Cache(Rache.from(baseCache))
```

Comment:
nit but think Rache.from(this.core.globalCache) is cleaner in both cases, ie twice that

Reply:
If we do `Rache.from(this.core.globalCache)` twice, then the flow is different when `this.core.globalCache` isn't set.

Currently, the `keyCache` and `nodeCache` share the same rache for that case. Doing Rache.from(...) twice means they each use a different rache.

I'm fine with either approach: either I'll add a comment to explain and leave as-is, or I change to use Rache.from twice